.spi.yml

```diff
@@ -0,0 +1,4 @@
+version: 1
+builder:
+  configs:
+    - documentation_targets: [MLX, MLXRandom, MLXNN, MLXOptimizers, MLXFFT, MLXLinalg]
```

Comment:
This tells swift package index how to build our docs.

See https://swiftpackageindex.com/SwiftPackageIndex/SPIManifest/1.2.0/documentation/spimanifest/commonusecases